src/component/feed_popup.rs

```diff
@@ -0,0 +1,47 @@
+use leptos::{ev::MouseEvent, *};
+
+use crate::component::connect::ConnectLogin;
+
+#[component]
+pub fn FeedPopUp<F: Fn(MouseEvent) + 'static>(
```

Comment:
Consider re-using the overlay component 

https://github.com/go-bazzinga/hot-or-not-web-leptos-ssr/blob/main/src/component/overlay.rs

lmk if its not usable here

Reply:
it's not allowing to scroll the feed